src/scenes/CollectionGalleryPage/CollectionGalleryPage.tsx

```diff
@@ -0,0 +1,88 @@
+import breakpoints, { pageGutter, size } from 'components/core/breakpoints';
+import Page from 'components/core/Page/Page';
+import styled from 'styled-components';
+import detectMobileDevice from 'utils/detectMobileDevice';
+import { useToastActions } from 'contexts/toast/ToastContext';
+import { useCallback, useEffect, useMemo } from 'react';
+import usePersistedState from 'hooks/usePersistedState';
+import { useBreakpoint } from 'hooks/useWindowSize';
+import usePrevious from 'hooks/usePrevious';
+import Head from 'next/head';
+import CollectionGallery from './CollectionGallery';
+
+// suggest mobile users to use landscape mode for 3-column experience.
+// use localStorage to prevent showing the toast again.
+function useSuggestion() {
```

Comment:
we can remove this! no longer used anymore: https://github.com/gallery-so/gallery/pull/538/files#diff-121f304ab5e398e8f28b52e3657b0a12c10d9d0de6ee320a3825a23decbee7e0